spec/CloudCodeLogger.spec.js

```diff
@@ -3,55 +3,55 @@ var LoggerController = require('../src/Controllers/LoggerController').LoggerCont
 var FileLoggerAdapter = require('../src/Adapters/Logger/FileLoggerAdapter').FileLoggerAdapter;
 
 describe("Cloud Code Logger", () => {
```

Comment:
just outta curiosity, why the change here?  it looks to me like you are just swapping the order of two tests?  or am i missing something else...